spec/core/elastic_connector_actions_spec.rb

```diff
@@ -601,165 +601,165 @@ def filter_validation_result(domain, state, errors)
-          :connector => {
-            :id => connector_id,
-            :filtering => anything
-          }
-        ),
-        :refresh => true
-      )
-
-      described_class.claim_job(connector_id)
-    end
-
-    it 're-reads a record in jobs index' do
-      expect(es_client).to receive(:get).with(
-        :index => jobs_index,
-        :id => job_id,
-        :ignore => 404
-      )
-
-      described_class.claim_job(connector_id)
-    end
-
-    context 'when connector is already syncing' do
-      before(:each) do
-        allow(es_client).to receive(:get)
-          .with(:index => connectors_index, :id => connector_id, :refresh => true, :ignore => 404)
-          .and_return(
-            { '_seq_no' => seq_no,
-              '_primary_term' => primary_term,
-              '_source' => {
-                'last_sync_status' => Connectors::SyncStatus::IN_PROGRESS
-              } }
-          )
-      end
-      it 'raises an error of specific type' do
-        expect { described_class.claim_job(connector_id) }.to raise_error(Core::JobAlreadyRunningError)
-      end
-    end
-
-    context 'when connector has changed version' do
-      before(:each) do
-        allow(es_client).to receive(:update)
-          .with(anything)
-          .and_raise(Core::ConnectorVersionChangedError.new(connector_id, seq_no, primary_term))
-      end
-      it 'raises an error of specific type' do
-        expect { described_class.claim_job(connector_id) }
-          .to raise_error(Core::ConnectorVersionChangedError)
-      end
-    end
-
-    context 'filtering rules' do
-      let(:connector_filtering) do
-        {
-          'domain' => 'default',
-          'active' => {
-            'rules' => [],
-            'advanced_snippet' => {},
-            'validation' => {}
-          },
-          'draft' => {
-            'rules' => [],
-            'advanced_snippet' => {},
-            'validation' => {}
-          }
-        }
-      end
-
-      let(:job_filtering) do
-        {
-          'domain' => 'default',
-          'rules' => [],
-          'advanced_snippet' => {},
-          'warnings' => []
-        }
-      end
-
-      before(:each) do
-        allow(es_client).to receive(:get)
-          .with(:index => connectors_index, :id => connector_id, :refresh => true, :ignore => 404)
-          .and_return(
-            { '_seq_no' => seq_no,
-              '_primary_term' => primary_term,
-              '_source' => {
-                'last_sync_status' => nil,
-                'filtering' => connector_filtering
-              } }
-          )
-      end
-
-      it 'has filtering rules' do
-        expect(es_client).to receive(:index).with(
-          :index => jobs_index,
-          :body => hash_including(:connector => hash_including(:filtering => [job_filtering])),
-          :refresh => true
-        )
-        described_class.claim_job(connector_id)
-      end
-    end
-  end
+  # describe '#claim_job' do
```

Comment:
As discussed with @vidok, these tests will be returned back after this PR is merged.